Update render-grid-in-template.md (#5246)
umbraco/UmbracoDocs: 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/grid-layout/render-grid-in-template.md

or alternatively you can provide the path to where the file resides:

```csharp

> @Html.GetGridHtml(Model, "propertyAlias", "mycustomrenderer")

Reviewer: Taking away the path from this example, makes it identical to the example above.
Let's keep a path in this example - and then use one of the ones you've found to be working instead.

What do you think?

```suggestion
@Html.GetGridHtml(Model, "propertyAlias", "/Views/Partials/grid/mycustomrenderer.cshtml")
```

Author: @sofietoft 
The suggestion is good but this won't work.
when we add path in the `@Html.GetGridHtml(Model, "propertyAlias", "/Views/Partials/grid/mycustomrenderer.cshtml") ` this will cause issue and trying to find view in /grid/ and added path in the code block. so this won't work.

Reviewer: Ah, alright. So is there are path we can add here that works?
Or is it because adding a path in this code doesn't work? 🤔

Reviewer: If it doesn't work to include the path in this code, it should be taken out.
I don't think it makes sense to have the same code example listed twice.
I also already mentions in line 19 where you should place the file, right?

Perhaps the only extra thing we need here, is the list of other potential places to put the file?

```

> And you can create a custom view in the `Views/Shared/grid/mycustomrenderer.cshtml` path. Possible other paths where we can create Grid layout views:
> `/Views/grid/mycustomrenderer.cshtml`
> `/Views/Shared/grid/mycustomrenderer.cshtml`
> `/Views/Partials/grid/mycustomrenderer.cshtml`
> `/Views/MacroPartials/grid/mycustomrenderer.cshtml`
> `/Views/Render/grid/mycustomrenderer.cshtml`
> `/Views/Shared/grid/mycustomrenderer.cshtml`
> `/Pages/Shared/grid/mycustomrenderer.cshtml`

Reviewer: ```suggestion
Possible paths where you can add the custom Grid layout views:

```none
/Views/grid/mycustomrenderer.cshtml
/Views/Shared/grid/mycustomrenderer.cshtml
/Views/Partials/grid/mycustomrenderer.cshtml
/Views/MacroPartials/grid/mycustomrenderer.cshtml
/Views/Render/grid/mycustomrenderer.cshtml
/Views/Shared/grid/mycustomrenderer.cshtml
/Pages/Shared/grid/mycustomrenderer.cshtml
```
